### PR TITLE
Put vars in vars file, remove unneeded replacements

### PR DESCRIPTION
**Pull request by sharms**

### CODE REVIEW

File: varsfiles/production.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+zone-transfer-ips: (( concat terraform_outputs.public_subnet_az1_cidr ", " terraform_outputs.public_subnet_az2_cidr ))
+master-ips: (( join "; " terraform_outputs.ENVIRONMENT_dns_private_ips ))
```
Comment (LinuxBozo) [inline varsfiles/production.yml]: If you are going through and doing separate varsfiles like this, no need for the ENVIRONMENT bit, as `sed` should be yanked from processing script as mentioned earlier.

File: ci/terraform-secrets.sh [hunk 2]
```diff
@@ -5,7 +5,7 @@ set -eux
 sed -i "s/ENVIRONMENT/${ENVIRONMENT}/g" pdns-config/varsfiles/terraform.yml
```
Comment (LinuxBozo) [inline ci/terraform-secrets.sh]: The terraform.yml file will no longer exist, so this needs yanked.

File: varsfiles/staging.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+zone-transfer-ips: (( concat terraform_outputs.public_subnet_az1_cidr ", " terraform_outputs.public_subnet_az2_cidr ))
+master-ips: (( join "; " terraform_outputs.ENVIRONMENT_dns_private_ips ))
```
Comment (LinuxBozo) [inline varsfiles/staging.yml]: If you are going through and doing separate varsfiles like this, no need for the ENVIRONMENT bit, as `sed` should be yanked from processing script as mentioned earlier.

**Review by LinuxBozo — CHANGES_REQUESTED** [review]

**Comment by sharms** [comment]
This PR has been updated to address comments @LinuxBozo

**Review by LinuxBozo — APPROVED** [review]

**Pull request merged**